### DISCUSSION TITLE
Can auto-budgets adjust when allocating transactions to previous month budget

### DISCUSSION THREAD

**Discussion by marcroberts** [firefly-iii/firefly-iii#6606] [Firefly III]
I've been using Firefly for just over a month and it's simplified many things we were doing with spreadsheets before.

I've been rather busy and had not finished assigning October's transactions to budgets before the end of the month (we use 'Add an amount every period' for most of our budgets). When I add an October transaction to a budget, it is correctly adjusting the October budget but the figure for November is not being adjusted to reflect the final amount remaining at the end of October before the November amount is added.

Is there a way to recalculate these once I've finished catching up on the October transactions?

This is likely to be a minor problem going forward even if I am up to date as our bank transactions seem to be delayed by a day or two before they appear via Nordigen so there will always be some transactions from the previous month that will need to be allocated after the month end.

**Comment by JC5** [comment]
> Is there a way to recalculate these once I've finished catching up on the October transactions?

You can remove the amount set using the dropdown next to the amount on the `/budgets` page. Then, run the cron job again. Use the extra argument `--date=2022-11-01`. That should recalculate them.
